### DISCUSSION TITLE
Q: 请问自定义表情支持SVG格式吗？ || Q: Does custom emoticon support SVG format?

### DISCUSSION THREAD

**Discussion by Tseshongfeeshur** [walinejs/waline#2057] [Q&A] [answered]
### Describe the bug | 问题描述

想给自定义表情使用SVG格式的图片，但不清楚是否支持。

### Website URL | 问题网站

NAN

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

I want to use images in SVG format for custom emoticons, but I don't know if it is supported.

### Website URL | Problem Website

NAN

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is your comment data store?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment, marked as answer]
支持 https://waline.js.org/cookbook/customize/emoji.html

---
Support https://waline.js.org/cookbook/customize/emoji.html
